Add dependency of __init__.py

Tasks must be done after exec of __init__, which creates the
src/_generated directory that tasks are based on.

Signed-off-by: Dongxiao Xu <dongxiao.xu@intel.com>

Upstream-Status: Submitted
(However it seems that this project is out of maintanence.)

diff -ruN telepathy-python-0.15.19-orig/src/Makefile.am telepathy-python-0.15.19/src/Makefile.am
--- telepathy-python-0.15.19-orig/src/Makefile.am	2011-03-10 08:51:49.000000000 +0800
+++ telepathy-python-0.15.19/src/Makefile.am	2011-03-10 08:54:45.000000000 +0800
@@ -39,17 +39,17 @@
 XSLTPROC_OPTS = --nonet --novalid --xinclude
 tools_dir = $(top_srcdir)/tools
 
-_generated/interfaces.py: $(tools_dir)/python-interfaces-generator.xsl $(wildcard $(spec_dir)/*.xml)
+_generated/interfaces.py: _generated/__init__.py $(tools_dir)/python-interfaces-generator.xsl $(wildcard $(spec_dir)/*.xml)
 	$(AM_V_GEN)$(XSLTPROC) $(XSLTPROC_OPTS) -o $@ \
 	    $(tools_dir)/python-interfaces-generator.xsl \
 	    $(spec_dir)/all.xml
 
-_generated/constants.py: $(tools_dir)/python-constants-generator.xsl $(wildcard $(spec_dir)/*.xml)
+_generated/constants.py: _generated/__init__.py $(tools_dir)/python-constants-generator.xsl $(wildcard $(spec_dir)/*.xml)
 	$(AM_V_GEN)$(XSLTPROC) $(XSLTPROC_OPTS) -o $@ \
 	    $(tools_dir)/python-constants-generator.xsl \
 	    $(spec_dir)/all.xml
 
-_generated/errors.py: $(tools_dir)/python-errors-generator.xsl $(wildcard $(spec_dir)/*.xml)
+_generated/errors.py: _generated/__init__.py $(tools_dir)/python-errors-generator.xsl $(wildcard $(spec_dir)/*.xml)
 	$(AM_V_GEN)$(XSLTPROC) $(XSLTPROC_OPTS) -o $@ \
 	    $(tools_dir)/python-errors-generator.xsl \
 	    $(spec_dir)/all.xml
@@ -58,7 +58,7 @@
 	$(AM_V_GEN)$(mkdir_p) $(dir $@)
 	@echo "# Placeholder for package" > $@
 
-_generated/%.py: $(tools_dir)/spec-to-python.xsl $(spec_dir)/%.xml
+_generated/%.py: _generated/__init__.py $(tools_dir)/spec-to-python.xsl $(spec_dir)/%.xml
 	$(AM_V_GEN)$(XSLTPROC) $(XSLTPROC_OPTS) -o $@ \
 	    $(tools_dir)/spec-to-python.xsl \
 	    $(spec_dir)/$*.xml
